tshirts.py

```diff
@@ -9,6 +9,10 @@ def size(cms):
 
 
 assert(size(37) == 'S')
+assert(size(38) == 'S')

```

Comment:
Size 38 should be classified as 'S' and 'M', Also for 42?
Size 38 can be either S or M not both. The exact expected outcome should be defined here.